coral/src/app/features/approvals/topics/components/TopicApprovalsTable.test.tsx

```diff
@@ -167,7 +172,12 @@ describe("TopicApprovalsTable", () => {
   describe("renders all content based on the column definition", () => {
     beforeAll(() => {
       mockIntersectionObserver();
-      render(<TopicApprovalsTable requests={mockedRequests} />);
+      render(
+        <TopicApprovalsTable
+          setDetailsModal={() => null}
```

Comment:
I would add a test case for the `setDetailsModal` to be called with the right value when the user clicks the overview button. It's within the responsibility of the component, so I would test it explicitly here.

Reply:
[9a686a5](https://github.com/aiven/klaw/pull/669/commits/9a686a54c7bd4691524e2471d8f02acca6755109)